app/assets/stylesheets/application.css.scss

```diff
@@ -143,11 +144,20 @@ img {
   padding: 10px 20px 10px;
   border: 4px solid $card-border-color;
   border-radius: 8px 24px;
+  box-shadow: 3px 5px 5px -3px #80808030;
+  transition: all 300ms;
+}
+
+.card-display:hover {
+  transform: scale(1.025);
+  background: $card-bg-color-hover;
 }
```

Comment:
The hover display is somewhat experimental, but I think it makes the website more dynamic and "enjoyable" to use.  Let me know what you think!

Here is a quick guide for transitions and transformations:
https://thoughtbot.com/blog/transitions-and-transforms